core/bones/spatial.py

```diff
@@ -150,7 +152,9 @@ def getEmptyValue(self) -> Tuple[float, float]:
         """
         return 0.0, 0.0
 
-    def singleValueFromClient(self, value: Dict, skel: str, name: str, origData: Dict):
+    def singleValueFromClient(self, value: Any, skel: 'SkeletonInstance',
+                              bone_name: str, client_data: dict
+                              ) -> tuple[Any, list[ReadFromClientError] | None]:
```

Comment:
Is it necessary to provide full type annotations in sub-classes as well?
I would like to keep maintenance overhead low, if possible.